package.json

```diff
@@ -21,12 +21,13 @@
     "jest-environment-jsdom": "29.7.0",
     "lint-staged": "15.3.0",
     "prettier": "3.4.2",
-    "release-it": "17.11.0",
+    "react": "19.0.0",
```

Comment:
npm throws a fit if this isn't here because it tries to grab the lowest compatible version of react instead of the highest